csds-client/client/v2/client.go

```diff
@@ -4,27 +4,33 @@ package client
 import (
 	"context"
 	"crypto/x509"
+	"encoding/json"
 	"envoy-tools/csds-client/client"
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
+	"path/filepath"
 	"strings"
 	"time"
 
 	csdspb_v2 "github.com/envoyproxy/go-control-plane/envoy/service/status/v2"
-	envoy_type_matcher "github.com/envoyproxy/go-control-plane/envoy/type/matcher"
+	envoy_type_matcher_v2 "github.com/envoyproxy/go-control-plane/envoy/type/matcher"
```

Comment:
You don't strictly need to have "v2" in aliases as usage of v2 is implied by this package, so everything here would be v2 by default. Leaving it up to you.

Reply:
Maybe I would like to keep it as `envoy_type_matcher`, or people may not realize that this package is also related to the version. In fact, when I refactored the code, I didn't realize it, so I put all the code related to it in the util.go. It was not until I added the support of v3 that I found out that this package is also related to the version.